app/views/admin/words/index.slim

```diff
@@ -0,0 +1,20 @@
+= link_to "New Word", new_admin_word_path, class:"btn btn-primary"
+table.table
+  thead
+    tr
+      th scope="col"  No.
+      th scope="col"  English
+      th scope="col"  VietName
+      th scope="col"  Description
+      th scope="col"  Action
+  tbody
+    - @words.each do |word|
+      tr
+        td = word.id
+        td = word.en_word
+        td = word.vi_word
+        td = word.description
+        td
+          = link_to "edit", edit_admin_word_path(word), class: "btn btn-warning mr-2"
+          = link_to 'delete', admin_word_path(word), method:"delete", data: {:confirm => "You sure?" }, class: "btn btn-danger"
```

Comment:
sau này ko đc dùng dấu mũi tên đối vs symbol, chỉ đc dùng :, ví dụ confirm: "You sure?"
trừ 1 vài trường hợp key của hash là string mới đc dùng, "1"=> "abc"